<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditView</class>
 <widget class="QWidget" name="EditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	color: #FFFFFF;
	border: 2px solid rgb(24, 204, 192);
	background-color: rgb(20,196,188);
}
QPushButton:hover{
	background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter#splitter::handle{
	background-color: rgb(20,196,188);
}
QSplitter#splitter::margin{
	background-color: rgb(20,196,188);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widgetQues" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#widgetQues{
	background-color: #FFFFFF;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelQues">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	border: 2px solid rgb(26, 222, 209);
	color: #FFFFFF;
	background-color: rgb(26, 222, 209);
}</string>
           </property>
           <property name="text">
            <string>试卷</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">QScrollArea{
	border: none;
}</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>291</width>
              <height>640</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitterRight">
      <property name="styleSheet">
       <string notr="true">QSplitter#splitterRight::handle{
	background-color: rgb(20,196,188);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widgetInfo" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#widgetInfo{
         background-color: #FFFFFF;
 }</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="layoutInfo" stretch="0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelInfo">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	border: 2px solid rgb(26, 222, 209);
	color: #FFFFFF;
	background-color: rgb(26, 222, 209);
}</string>
            </property>
            <property name="text">
             <string>信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widgetInfo2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelInfoText">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetConf" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#widgetConf{
         background-color: #FFFFFF;
 }</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelConf">
          <property name="styleSheet">
           <string notr="true">QLabel{
	border: 2px solid rgb(26, 222, 209);
	color: #FFFFFF;
	background-color: rgb(26, 222, 209);
}</string>
          </property>
          <property name="text">
           <string>发放配置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetConf2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <spacer name="verSpacerConf">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetAct" native="true">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#widgetAct{
         background-color: #FFFFFF;
 }</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelAct">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	border: 2px solid rgb(26, 222, 209);
	color: #FFFFFF;
	background-color: rgb(26, 222, 209);
}</string>
          </property>
          <property name="text">
           <string>操作</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetAct2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>12</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnAdd">
             <property name="text">
              <string>添加题目</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnPush">
             <property name="text">
              <string>发放试卷</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verSpacerAct">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
